pydis_site/apps/api/models/bot/infraction.py

```diff
@@ -15,7 +15,8 @@ class Infraction(ModelReprMixin, models.Model):
         ("mute", "Mute"),
         ("kick", "Kick"),
         ("ban", "Ban"),
-        ("superstar", "Superstar")
+        ("superstar", "Superstar"),
+        ("voice_ban", "Voice Ban")
```

Comment:
Maybe we should put a trailing comma here to stop diffs like this in future
```suggestion
        ("voice_ban", "Voice Ban"),
```